configmap/parse.go

```diff
@@ -166,14 +166,14 @@ func AsDuration(key string, target *time.Duration) ParseFunc {
 }
 
 // AsStringSet parses the value at key as a sets.String (split by ',') into the target, if it exists.
```

Comment:
This one needs change to sets.String -> sets.Set[String]

Reply:
I think there are more:

sets.String -> sets.Set[String]
https://github.com/knative/pkg/blob/71979fdc8b21d241a5d1a582872a074d0a6417f6/test/spoof/response_checks.go#L66
https://github.com/knative/pkg/blob/71979fdc8b21d241a5d1a582872a074d0a6417f6/leaderelection/context_test.go#L48
https://github.com/knative/pkg/blob/71979fdc8b21d241a5d1a582872a074d0a6417f6/leaderelection/context_test.go#L181

Reply:
Fixed